### ISSUE TITLE
VSTest has a -preview indication

### ISSUE THREAD

**Issue by MichelZ** [actions/virtual-environments#5971]
### Description

When running tests on Azure Devops, Windows Server 2022 today (2022-07-26), it looks like a preview version of Visual Studio's vstest is used:

```
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\Extensions\TestPlatform\vstest.console.exe "@D:\a\_temp\syvwbhqrxhn.tmp"
Microsoft (R) Test Execution Command Line Tool Version 17.2.0-preview-20220401-07 (x64)
```

Instead of the preview version, it should use the release version?

### Platforms affected

- [X] Azure DevOps
- [ ] GitHub Actions

### Virtual environments affected

- [ ] Ubuntu 18.04
- [ ] Ubuntu 20.04
- [ ] Ubuntu 22.04
- [ ] macOS 10.15
- [ ] macOS 11
- [ ] macOS 12
- [ ] Windows Server 2019
- [X] Windows Server 2022

### Image version and build link

```
Starting: Initialize job
Agent name: 'Hosted Agent'
Agent machine name: 'fv-az290-973'
Current agent version: '2.206.1'
Operating System
Virtual Environment
Environment: windows-2022
Version: 20220710.1
Included Software: https://github.com/actions/virtual-environments/blob/win22/20220710.1/images/win/Windows2022-Readme.md
Image Release: https://github.com/actions/virtual-environments/releases/tag/win22%2F20220710.1
Virtual Environment Provisioner
1.0.0.0-main-20220701-2
Current image version: '20220710.1'
Agent running as: 'VssAdministrator'
Prepare build directory.
Set build variables.
Download all required tasks.
Downloading task: UseDotNet (2.206.0)
Downloading task: DownloadPipelineArtifact (2.198.0)
Downloading task: CopyFiles (2.200.0)
Downloading task: VSTest (2.205.0)
Checking job knob settings.
   Knob: AgentToolsDirectory = C:\hostedtoolcache\windows Source: ${AGENT_TOOLSDIRECTORY} 
   Knob: AgentPerflog = c:\vsts\perflog Source: ${VSTS_AGENT_PERFLOG} 
Finished checking job knob settings.
Start tracking orphan processes.
Finishing: Initialize job
```

### Is it regression?

No

### Expected behavior

Uses a release build of vstest

### Actual behavior

Uses a preview version of vstest

### Repro steps

Execute a vstest task

```
- task: VSTest@2
  displayName: 'Run integration tests (.NET FX)'
  condition: and(succeeded(), eq(variables['RunTests'], 'true'))
  inputs:
    testSelector: 'testAssemblies'
    testAssemblyVer2: '**\Microsoft.Identity.Test.Integration.netfx\bin\**\net4*\Microsoft.Identity.Test.Integration.NetFx.dll'
    searchFolder: '$(System.DefaultWorkingDirectory)'
    rerunFailedTests: true
    rerunMaxAttempts: '3'
    runInParallel: false
    codeCoverageEnabled: true
```

**Comment by al-cheb** [comment]
Hey @MichelZ.
We will check.

**Comment by al-cheb** [comment]
@MichelZ, As I can see there are no available vstest release versions  - https://github.com/microsoft/vstest-docs/blob/main/docs/releases.md#1720

17.2.0 marks as https://github.com/microsoft/vstest/compare/v17.2.0-preview-20220401-07...v17.2.0
![image](https://user-images.githubusercontent.com/47745270/180950300-a7c8fda0-01b9-4737-a5dc-523cb4041789.png)

**Comment by MichelZ** [comment]
Something is weird....

https://www.nuget.org/packages/Microsoft.TestPlatform/#versions-body-tab
It clearly says there is a non-preview 17.2.0 version.

![image](https://user-images.githubusercontent.com/1040252/180956165-7c90c72a-890c-489c-85ec-8b16b6402ec1.png)

The changes seem to be insignificant though, and probably not related to the issue I was investigating.
It just struck me as odd that it said -preview, and I thought it makes sense to use a non-preview build.

**Comment by al-cheb** [comment]
I  have got a response from the test platform team:

> That’s actually the release version- there was a branding bug there. Unless the branding is affecting a lot of customers negatively we don’t plan to service it. It’s fixed in 17.3.

**Comment by MichelZ** [comment]
Thanks for checking, very much appreciated
